stable/datadog/README.md

```diff
@@ -1,177 +1,131 @@
-
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+As a best practice, a YAML file that specifies the values for the chart parameters should be provided to configure the chart: 
+
+1.  **Copy the default [`datadog-values.yaml`](/values.yaml) value file.**
+2.  Set the `apiKey` parameter with your [Datadog API key](https://app.datadoghq.com/account/settings#api).
+3.  Upgrade the Datadog Helm chart with the new `datadog-values.yaml` file:
 
 ```bash
-helm install --name my-release \
-  --set datadog.apiKey=YOUR-KEY-HERE,datadog.logLevel=DEBUG \
-  stable/datadog
+helm upgrade -f datadog-values.yaml <RELEASE_NAME> stable/datadog --recreate-pods
+```
+
+See the [All configuration options](#all-configuration-options) section to discover all possibilities offered by the Datadog chart.
+
+### Enabling Log Collection
+
+Update your [datadog-values.yaml](/values.yaml) file with the following log collection configuration:
+
+```
+datadog:
+  (...)
+ logsEnabled: true
+ logsConfigContainerCollectAll: true
 ```
 
-Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+then upgrade your Datadog Helm chart:
 
 ```bash
-helm install --name my-release -f my-values.yaml stable/datadog
+helm upgrade -f datadog-values.yaml <RELEASE_NAME> stable/datadog --recreate-pods
 ```
 
-**Tip**: You can copy and customize the default [values.yaml](values.yaml)
+### Enabling Process Collection
 
-### Image repository and tag
+Update your [datadog-values.yaml](/values.yaml) file with the process collection configuration: 
 
-Datadog [offers two variants](https://hub.docker.com/r/datadog/agent/tags/), switch to a `-jmx` tag if you need to run JMX/java integrations. The chart also supports running [the standalone dogstatsd image](https://hub.docker.com/r/datadog/dogstatsd/tags/).
+```
+datadog:
+  (...)
+  processAgentEnabled: true
+```
 
-Starting with version 1.0.0, this chart does not support deploying Agent 5.x anymore. If you cannot upgrade to Agent 6.x, you can use a previous version of the chart by calling helm install with `--version 0.18.0`.
+then upgrade your Datadog Helm chart:
 
-### DaemonSet and Deployment
+```bash
+helm upgrade -f datadog-values.yaml <RELEASE_NAME> stable/datadog --recreate-pods
+```
 
-By default, the Datadog Agent runs in a DaemonSet. It can alternatively run inside a Deployment for special use cases.
+### Kubernetes event collection
 
-**Note:** simultaneous DaemonSet + Deployment installation within a single release will be deprecated in a future version, requiring two releases to achieve this.
+To enable event collection, set the `datadog.leaderElection`, `datadog.collectEvents` and `rbac.create` options to `true`.
 
-### Secret
+It is now recommended to use the Datadog Cluster Agent to collect the events - Refer to the [Enabling the Datadog Cluster Agent](#enabling-the-datadog-cluster-agent) section.
```

Comment:
Should this be moved at the top of the section instead, as that's what we recommend?
Then the `datadog.collectEvents` can be introduced with "alternatively, you can set event collection on the daemonset with..."